### PR TITLE
Pin kubernetes 1.17.0 and switch to go.mod

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by deads2k** [comment]
This change looks wrong.  I expected to see the k8s.io/* repos in the "normal" section on tag `v0.17.0`

**Comment by deads2k** [comment]
/hold

this should be set to tags

File: go.mod [hunk 1]
```diff
@@ -52,15 +52,28 @@ require (
...
 	k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
 	sigs.k8s.io/yaml v1.1.0
 	vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787
 )
+
+replace (
+	github.com/ghodss/yaml => github.com/ghodss/yaml v0.0.0-20170327235444-0ca9ea5df545
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20191217141120-791af96035a5
+	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20191216194936-57f413491e9e
+	k8s.io/api => k8s.io/api v0.17.0
```
Comment (deads2k) [inline go.mod]: no replaces.  This breaks transitive dep calculations

**Comment by mfojtik** [comment]
/hold cancel

Removed replace, moved to to go 1.13, this is ready to go.

**Pull request merged**